styleguide/samples/_advice_collection_advisernet.html.haml

```diff
@@ -1,39 +0,0 @@
-.cads-advisernet
```

Comment:
Removed this sample on the rationale that as part of the upcoming theming work we'll shift advisernet specific theming into the website repo